### PR TITLE
feat: display annotated sentences

### PR DESCRIPTION
**Pull request by deltork**
### PR Goal? 
display manually annotated read-along

### Feedback sought? 
UI and format check

### Priority? 
by July 8th

### Tests added? 
yes

### How to test? 
spin up web-component `npx nx dev web-component` and visit http://localhost:8941/ej-fra/index-annotated.html

See test-data/ej-fra/ej-fra-annotated.read long formating 

### Confidence? 
high

### Version change? 
minor

### CODE REVIEW

**Comment by joanise** [comment]
This PR is marked as a draft PR, yet you asked for review. Is it ready?

File: packages/web-component/src/components/read-along-component/read-along.tsx [hunk 1]
```diff
@@ -1046,7 +1077,31 @@ export class ReadAlongComponent {
...
-
+      /**
+       * parse defined annotations information from the .readalong meta
+       * annotations-id defines the id of each layer
+       * annotations-labels defines the display label for each layer
+       * annotations-label-{locale} defines the localized display label for each layer
+       * all lists are delimeterd by comma
+       */
+      if (this.meta["annotations-ids"]) {
+        const delimeter = ",";
```
Comment (joanise) [inline packages/web-component/src/components/read-along-component/read-along.tsx]: Typo: should be `delimiter` with an `i` in the third syllable

File: packages/web-component/src/i18n/messages.spa.json [hunk 2]
```diff
@@ -26,6 +26,7 @@
     "eng": "Inglés",
     "fra": "Francés",
     "spa": "Español",
-    "close": "Cerrar"
+    "close": "Cerrar",
+    "annotations-tooltip": "Annotations"
```
Comment (joanise) [inline packages/web-component/src/i18n/messages.spa.json]: `"Anotaciones"`

**Review by joanise — CHANGES_REQUESTED** [review]
Generally looks good and works well.

One thing: Aidan suggested "removing" instead of just "hiding" the hidden layers, for better rendering, and I agree it would be worth making this change.

**Comment by deltork** [comment]
> Generally looks good and works well.
> 
> One thing: Aidan suggested "removing" instead of just "hiding" the hidden layers, for better rendering, and I agree it would be worth making this change.

I have added that now. Thanks for the reminder!

**Review by joanise — APPROVED** [review]
This looks good now.

**Pull request merged**